magenta-lib/src/test/scala/magenta/input/resolver/DeploymentTypeResolverTest.scala

```diff
@@ -95,7 +95,7 @@ class DeploymentTypeResolverTest
     val deploymentTypesWithParams = List(
       stubDeploymentType(
         Seq("upload", "deploy"),
-        register => List(Param[String]("param1")(register))
+        implicit register => List(Param[String]("param1"))
```

Comment:
`register` is getting shifted around here because the constructor of `Param` now takes 3 implicit parameters, and it's easier to make `register` implicit rather than pass in 3 parameters.